src/VoxaEvent.ts

```diff
@@ -87,6 +90,44 @@ export abstract class VoxaEvent implements IVoxaEvent {
     this.initLogger(logOptions);
   }
 
+  public async getUserInformation(): Promise<any> {
+    if (_.has(this, "google")) {
+      return this.getUserInformationWithGoogle();
+    }
+
+    return await this.getUserInformationWithLWA();
+  }
+
+  public getUserInformationWithGoogle(): any {
+    const result: any = jwt.decode(this.user.idToken);
+    result.emailVerified = result.email_verified;
+    result.familyName = result.family_name;
+    result.givenName = result.given_name;
+
+    delete result.email_verified;
+    delete result.family_name;
+    delete result.given_name;
+
+    return result;
+  }
+
+  public async getUserInformationWithLWA(): Promise<any> {
```

Comment:
please create an interface for this return type, would be great if both methods share that BTW